modules/electrophysiology_browser/php/models/electrophysioevents.class.inc

```diff
@@ -20,34 +21,54 @@ class ElectrophysioEvents
      * Construct an Event object
      *
      * @param integer $physioFileID Electrophysiological file ID
-     *                              to collect annotation data from
+     *                              to collect event data from
      */
     function __construct(int $physioFileID)
     {
         $this->_physioFileID = $physioFileID;
         $db = \NDB_Factory::singleton()->database();
 
         $taskEvents = $db->pselect(
-            'SELECT te.* 
+            'SELECT te.*
                 FROM physiological_task_event AS te
                 JOIN physiological_event_file AS f
-                ON f.EventFileID = te.EventFileID 
+                ON f.EventFileID = te.EventFileID
                 WHERE f.PhysiologicalFileID=:PFID AND f.FileType="tsv"',
             ['PFID' => $this->_physioFileID]
         );
 
-        /**
-         * TODO: Get event params and metadata.
-         *       NOT in the scope of current task
-         **/
+        $taskEventIDs = array_map(
```

Comment:
Open question: do we need to check first that `$taskEvents` is not empty, or that the given `$physioFileID` exists in db? Let say we try to build an `electrophysioevents` with a `$physioFileID` that is set to a value that does not exist in db, is that an expected behaviour?